MoreLinq.Test/EndsWithTest.cs

```diff
@@ -61,19 +61,20 @@ public bool EndsWithWithStrings(string first, string second)
         [Test]
         public void EndsWithReturnsTrueIfBothEmpty()
         {
-            new int[0].EndsWith(new int[0]);
+            Assert.True(new int[0].EndsWith(new int[0]));
         }
 
         [Test]
         public void EndsWithReturnsFalseIfOnlyFirstIsEmpty()
         {
-            new int[0].EndsWith(new[] {1,2,3});
+            Assert.False(new int[0].EndsWith(new[] {1,2,3}));
         }
 
-        [Test]
-        public void EndsWithReturnsFalseIfOnlySecondIsEmpty()
+        [TestCase("", "", Result = true)]
+        [TestCase("1", "", Result = true)]
+        public bool EndsWithReturnsTrueIfSecondIsEmpty(string first, string second)
         {
-            new[] {1,2,3}.EndsWith(new int[0]);
+            return first.StartsWith(second);
```

Comment:
I reckon you meant to call `EndsWith` here instead. [My earlier comment on a `StartsWith` test not testing the right implementation](https://github.com/morelinq/MoreLINQ/pull/170/commits/948d15a1488b24afe9d3d7c549a7be13afb1715d#r62559356) applies here too.